docs/written/trove/lists.scrbl

```diff
@@ -1623,13 +1623,104 @@ end
 }
 
 @list-method["member"]
+
+Returns true if the @pyret{elt} is located somewhere in the list.
+
+@examples{
+check:
+  [list: 'a', 'b', 'c'].member('a') is true
```

Comment:
I think these are two separate things.  `f(x) is true` is stating something about `f`.  `x satisfies f` is stating something about `x`.  If the documentation is about `f`, I'd use the first version.

In general, sure, it's a stylistic nit to say `f(x) is true` when you could say `x satisfies f`, just as much so as `if x: true else y` is better written as `x or y`, etc...